README.md

```diff
@@ -1,2 +1,12 @@
 # website
 kubesphere website
+
+## Running the website locally
+
+hugo version: hugo_extended_0.70.0 + [hugo releases](https://github.com/gohugoio/hugo/releases)
+
+installed Hugo and clone this project locally. then run:
+
+```
+hugo server -D
+```
```

Comment:
Please add a brief introduction for how to contribute the i18n files, it would be better to add a `tr` folder for Turkish under each corresponding folder.

Reply:
ok